utils.py

```diff
@@ -2,15 +2,18 @@
 
 import time
 
-VERBOSE_OUTPUT = True
-TIME_SCALE = 1
+class Printer:
```

Comment:
Not very Pythonic.

Reply:
Agreed! This was the least dirty way I thought of that enables passing values from the run.py. The other way would be passing them in each `ui_print` call which I wanted to avoid.

I'll merge this since I guess we have bigger priorities than being Pythonic now, but definitely a valid point, do you have any ideas how to do this better?